src/domain/community/user/user.service.authorization.ts

```diff
@@ -77,9 +73,10 @@ export class UserAuthorizationService {
             user.authorization
           );
       }
+      user.preferences = preferences;
```

Comment:
I think the preferences do not need to inherit all of the user authorization but just the `user-self` for this resource.

Reply:
the model is that all parent authorizations are cascaded through, for now. We may indeed a more sophisticated model later - there is lots of options in that space: https://github.com/alkem-io/alkemio/issues/312